src/_gettsim/transfers/rente.py

```diff
@@ -491,19 +500,68 @@ def _ges_rente_altersgrenze_abschlagsfrei_mit_besond_langj(  # noqa: PLR0913
     return out
 
 
-def referenz_alter_abschlag(
+@dates_active(start="2018-01-01", change_name="_ges_rente_altersgrenze_abschlagsfrei")
+def _ges_rente_altersgrenze_abschlagsfrei_ohne_arbeitsl_frauen(
+    ges_rente_regelaltersgrenze: float,
+    _ges_rente_langj_altersgrenze: float,
+    _ges_rente_besond_langj_altersgrenze: float,
+    ges_rente_vorauss_regelrente: bool,
+    ges_rente_vorauss_langj: bool,
+    ges_rente_vorauss_besond_langj: bool,
+) -> float:
+    """Full retirement age.
+
+    Age at which pension can be claimed without deductions. This age is smaller or equal
+    to the normal retirement age (FRA<=NRA) and depends on personal characteristics as
+    gender, insurance duration, health/disability, employment status.
+
+    Parameters
+    ----------
+    ges_rente_regelaltersgrenze
+        See :func:`ges_rente_regelaltersgrenze`.
+    _ges_rente_langj_altersgrenze
+        See :func:`_ges_rente_langj_altersgrenze`.
+    _ges_rente_besond_langj_altersgrenze
+        See :func:`_ges_rente_besond_langj_altersgrenze`.
+    ges_rente_vorauss_regelrente
+        See :func:`ges_rente_vorauss_regelrente`.
+    ges_rente_vorauss_langj
+        See :func:`ges_rente_vorauss_langj`.
+    ges_rente_vorauss_besond_langj
+        See :func:`ges_rente_vorauss_besond_langj`.
+
+    Returns
+    -------
+    Full retirement age.
+
+    """
+
+    out = float("inf")
+    if ges_rente_vorauss_regelrente:
+        out = ges_rente_regelaltersgrenze
+    if ges_rente_vorauss_langj:
+        out = min([out, _ges_rente_langj_altersgrenze])
+    if ges_rente_vorauss_besond_langj:
+        out = min([out, _ges_rente_besond_langj_altersgrenze])
```

Comment:
Just my usual check whether this is being tested for sure? codecov flags it, but that might well be false.

Reply:
Ah sorry, I thought that codecov check passed => Test coverage of newly added / changed lines is 100%. I will have that in mind in the future.

I added some tests that cover these lines but only for one birth cohort (where the tests are particularly easy). I think it makes sense to properly test the pathways into retirement once we are rewriting them (in particular, the Rente für Frauen).

Reply:
Do we want to test for `out = float("Nan")` in `referenz_alter_abschlag`?

I think not because i) coding "no eligibility for early retirement" with NaNs as the early retirement age should change in the future and ii) as far as I understood, these tests check against the numbers implied by the law (and not such technical details).

If we want to test for it, I don't know how to include `float("NaN")` as expected output in the yaml file.